codefori/vscode-ibmi
# Fixed Component manager exportation

### Changes
The Component Manager could not be used by other extensions because of the way types were exported.
This PR fixes this, allowing other extension to actually contribute components to Code for i.

It also adds an `.npmrc` file to avoid issues when `npm` is ran with a proxy npm repo defined locally.

### How to test this PR
1. Run the test cases
2. Open connection settings
3. Check all Components are listed correctly

### Checklist
